deploy-manage/security/private-connectivity-azure.md

```diff
@@ -73,14 +73,15 @@ The process of setting up the private connection with Azure Private link is spli
 |  | 4. [Associate the Azure private connection policy with your deployments](#ec-associate-traffic-filter-private-link-rule-set). |
 |  | 5. [Interact with your deployments over Private Link](#ec-azure-access-the-deployment-over-private-link). |
 
+After you create your private connection policy, you can [edit](#ec-azure-edit-traffic-filter-private-link-rule-set), [disassociate](#remove-filter-deployment), or [delete](#ec-azure-delete-traffic-filter-private-link-rule-set) it.
 
 ### Create your private endpoint and DNS entries in Azure [ec-private-link-azure-dns]
 
 1. Create a private endpoint in your VNet using the alias for your region.
 
     Follow the [Azure instructions](https://docs.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal#create-a-private-endpoint) for details on creating a private endpoint to an endpoint service.
 
-    Use [the service aliases for your region](/deploy-manage/security/azure-private-link-traffic-filters.md#ec-private-link-azure-service-aliases). Select the **Connect to an Azure resource by resource ID or alias** option. For example for the region `eastus2` the service alias is `eastus2-prod-002-privatelink-service.64359fdd-7893-4215-9929-ece3287e1371.eastus2.azure.privatelinkservice`
+    Use [the service aliases for your region](#ec-private-link-azure-service-aliases). Select the **Connect to an Azure resource by resource ID or alias** option. For example for the region `eastus2` the service alias is `eastus2-prod-002-privatelink-service.64359fdd-7893-4215-9929-ece3287e1371.eastus2.azure.privatelinkservice`
```

Comment:
shouldn't we say #ec-private-connectivity-azure-service-aliases ?

Reply:
these are service aliases unique to "azure private link" so in this case we don't need an update.